From 741f5285ee7dedd86d5e6142d4f6a411c65667ad Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Thu, 14 Sep 2023 20:47:24 +0200
Subject: [PATCH] Drop unused KI18n header/dependency

Does not actually have translations.

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
(cherry picked from commit 62c8a809800576d0898da0e7f5941a4070637c58)
---
 CMakeLists.txt        | 3 +--
 ffmpegthumbnailer.cpp | 1 -
 tests/CMakeLists.txt  | 1 -
 3 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 806b703..46b9e13 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,7 +24,7 @@ include(FeatureSummary)
 include(ECMDeprecationSettings)
 
 find_package(Qt${QT_MAJOR_VERSION} ${QT_MIN_VERSION} CONFIG REQUIRED COMPONENTS Core Gui)
-find_package(KF${QT_MAJOR_VERSION} ${KF_MIN_VERSION} REQUIRED COMPONENTS KIO I18n Config)
+find_package(KF${QT_MAJOR_VERSION} ${KF_MIN_VERSION} REQUIRED COMPONENTS KIO Config)
 find_package(FFmpeg COMPONENTS AVCODEC AVFORMAT SWSCALE)
 
 include_directories(
@@ -71,7 +71,6 @@ target_link_libraries(ffmpegthumbs
     Qt::Gui
     KF${QT_MAJOR_VERSION}::KIOWidgets
     KF${QT_MAJOR_VERSION}::KIOCore
-    KF${QT_MAJOR_VERSION}::I18n
     KF${QT_MAJOR_VERSION}::ConfigCore
     KF${QT_MAJOR_VERSION}::ConfigGui
     ${AVUTIL_LIBRARIES}
diff --git a/ffmpegthumbnailer.cpp b/ffmpegthumbnailer.cpp
index 5bee237..40bf1be 100644
--- a/ffmpegthumbnailer.cpp
+++ b/ffmpegthumbnailer.cpp
@@ -11,7 +11,6 @@
 
 #include <limits>
 
-#include <KLocalizedString>
 #include <KPluginFactory>
 #include <QCheckBox>
 #include <QFormLayout>
diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 7f4c7e3..ec7e964 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -26,7 +26,6 @@ target_link_libraries(ffmpegthumbtest
     Qt::Gui
     KF${QT_MAJOR_VERSION}::KIOWidgets
     KF${QT_MAJOR_VERSION}::KIOCore
-    KF${QT_MAJOR_VERSION}::I18n
     KF${QT_MAJOR_VERSION}::ConfigCore
     KF${QT_MAJOR_VERSION}::ConfigGui
     ${AVUTIL_LIBRARIES}
-- 
2.42.0

